[rospy.client][INFO] 2020-11-17 08:40:46,947: init_node, name[/train_13514_1605602446943], pid[13514]
[xmlrpc][INFO] 2020-11-17 08:40:46,948: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-11-17 08:40:46,948: Started XML-RPC server [http://luigi:33141/]
[rospy.init][INFO] 2020-11-17 08:40:46,949: ROS Slave URI: [http://luigi:33141/]
[rospy.impl.masterslave][INFO] 2020-11-17 08:40:46,949: _ready: http://luigi:33141/
[rospy.registration][INFO] 2020-11-17 08:40:46,950: Registering with master node http://192.168.0.201:11311
[xmlrpc][INFO] 2020-11-17 08:40:46,950: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-11-17 08:40:47,049: registered with master
[rospy.rosout][INFO] 2020-11-17 08:40:47,049: initializing /rosout core topic
[rospy.rosout][INFO] 2020-11-17 08:40:47,054: connected to core topic /rosout
[rospy.simtime][INFO] 2020-11-17 08:40:47,056: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-11-17 08:40:47,076: topic[/odom] adding connection to [http://192.168.0.201:37487/], count 0
[rospy.internal][INFO] 2020-11-17 08:40:47,295: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-11-17 08:40:55,546: topic[/bev/flow_image] adding connection to [http://192.168.0.201:41591/], count 0
[rospy.internal][INFO] 2020-11-17 08:40:55,550: topic[/bev/occupancy_image] adding connection to [http://192.168.0.201:41591/], count 0
[rospy.core][INFO] 2020-11-17 08:44:14,762: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-11-17 08:44:14,768: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-11-17 08:44:14,770: topic[/bev/flow_image] removing connection to http://192.168.0.201:41591/
[rospy.internal][INFO] 2020-11-17 08:44:14,771: topic[/bev/occupancy_image] removing connection to http://192.168.0.201:41591/
[rospy.internal][INFO] 2020-11-17 08:44:14,772: topic[/odom] removing connection to http://192.168.0.201:37487/
[rospy.impl.masterslave][INFO] 2020-11-17 08:44:14,772: signal-2
[rospy.internal][WARNING] 2020-11-17 08:44:14,773: Unknown error initiating TCP/IP socket to 192.168.0.201:38831 (http://192.168.0.201:41591/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 563, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[rospy.internal][WARNING] 2020-11-17 08:44:14,775: Unknown error initiating TCP/IP socket to 192.168.0.201:38831 (http://192.168.0.201:41591/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 563, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[rospy.core][INFO] 2020-11-17 08:44:15,261: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-17 08:44:15,293: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-17 08:44:15,326: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-17 08:44:15,352: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-17 08:44:15,402: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-17 08:44:15,453: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-17 08:44:15,462: signal_shutdown [atexit]
